packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -130,6 +123,17 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    * @defaultvalue false
    */
   originFromZero?: boolean;
+
+  /**
+   * Optional flag to display tick marks on each step
+   */
+  showThumbTooltip?: boolean;
+
+  /**
+   * Optional flag to indicate whether to have tick marks
+   * @defaultvalue false
+   */
+  enableMarks?: boolean | ISliderMarksArrayFormat[];
```

Comment:
More comments on this prop now that it allows not only a boolean flag. Also, not a very good idea to mix this 2 different features in just one prop. This makes it very confusing to a developer on how to use it. At least the naming already should not refer to `enable` only state.

Reply:
ok just added some more comments and should I just change the name to "marks"?

Reply:
`marks` would be ok only if you decide to go with just one prop.

Reply:
done